tools/js-tools/package.json

```diff
@@ -21,6 +21,7 @@
 		"@eslint/json": "0.13.0",
 		"@octokit/auth-token": "5.1.2",
 		"@octokit/rest": "21.1.1",
+		"@stylistic/stylelint-plugin": "3.1.3",
```

Comment:
Do we need this as a direct dependency?

Reply:
In one iteration it seemed the case, but now it seems to work without: 06889a207bd